core/src/main/resources/application.yml

```diff
@@ -121,6 +121,7 @@ cb:
     url: http://localhost:8088
     contextPath: /environmentservice
   freeipa:
+    image.catalog.url: https://cloudbreak-imagecatalog.s3.amazonaws.com/v3-prod-freeipa-image-catalog.json
```

Comment:
Image catalogs are defined in profile specific application properties, like application-dev.yml, please remove this

Reply:
No, we should not move this to anywhere as overrides are going to be applied by environment specific helm configuration - as they are already in place for other image catalog urls. Tbh we should do the opposite of what you are suggesting with other image urls :-)

Reply:
That is also fine with me, just do it consistently